### PR TITLE
copy php from nixpkgs

### PR DESCRIPTION
**Pull request by phunehehe**
and add back 5.3 (though currently unused)

### CODE REVIEW

**Comment by phunehehe** [comment]
(don't merge yet some build failure is going on)

File: pkgs/php/fix-paths-php53.patch [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ 
+ 
+ if test "$PHP_GETTEXT" != "no"; then
+-  for i in $PHP_GETTEXT /usr/local /usr; do
+-    test -r $i/include/libintl.h && GETTEXT_DIR=$i && break
+-  done
++  GETTEXT_DIR=$PHP_GETTEXT
+ 
+   if test -z "$GETTEXT_DIR"; then
+     { echo "configure: error: Cannot locate header file libintl.h" 1>&2; exit 1; }
```
Comment (phunehehe) [inline pkgs/php/fix-paths-php53.patch:69]: This file was adapted from the more generic fix-paths.patch
The different in change locations is too big for the patch scanner.

**Comment by proger** [comment]
Could you add the sdk change into this PR as well?

**Comment by phunehehe** [comment]
@proger you mean, changing sdk to use php53 by default? we don't want that do we?

**Comment by proger** [comment]
No, https://github.com/zalora/microgram/pull/42#discussion_r40424250

**Comment by phunehehe** [comment]
yeah but that means `php = ugpkgs.php53;`

**Comment by proger** [comment]
This means that `sdk.php53` needs to be somehow available to the apps.

**Comment by phunehehe** [comment]
why don't I just leave this as is and use `php = ugpkgs.php53;` when I want to use php53?

**Comment by phunehehe** [comment]
merge?

**Pull request merged**